core/src/components/modal/test/can-dismiss/modal.spec.tsx

```diff
@@ -44,6 +43,7 @@ describe('modal: canDismiss', () => {
         components: [Modal],
         template: () => (
           <ion-modal
+            overlayIndex={1}
```

Comment:
Why do we need to set overlayIndex?

Reply:
We have it explicitly set as a required field: https://github.com/ionic-team/ionic-framework/blob/f143bd0a11941739ee616908891509b48bd48463/core/src/components/modal/modal.tsx#L103